en/micro-integrator/docs/references/synapse-properties/endpoint-properties.md

```diff
@@ -552,8 +552,45 @@ At any given time, the state of the endpoint can be one of the following. During
   </tr>
 </table>
 
+The following is the configuration for the address endpoint. Since we all are only interested in error configurations, the same applies for WSDL endpoints as well. The error handling configuration are as follows:
```

Comment:
Do not use "we". How about "In this scenario, you can apply the same configuration for WSDL endpoints to create error configurations." Does it give the same meaning?

Reply:
How about this?

"Note that you can apply the same configuration for WSDL endpoints to create error configurations as well."

Reply:
Ok